software/winrm/src/main/java/org/apache/brooklyn/location/winrm/WinRmMachineLocation.java

```diff
@@ -80,6 +81,9 @@
     public static final ConfigKey<String> OPERATION_TIMEOUT = newConfigKeyWithPrefix(BrooklynConfigKeys.BROOKLYN_WINRM_CONFIG_KEY_PREFIX, WinRmTool.OPERATION_TIMEOUT);
     public static final ConfigKey<Integer> RETRIES_OF_NETWORK_FAILURES = newConfigKeyWithPrefix(BrooklynConfigKeys.BROOKLYN_WINRM_CONFIG_KEY_PREFIX, WinRmTool.RETRIES_OF_NETWORK_FAILURES);
     public static final ConfigKey<Boolean> USE_HTTPS_WINRM = WinRmTool.USE_HTTPS_WINRM;
+    public static final ConfigKey<String> VERIFY_WINDOWS_UP = newStringConfigKey("verifyWindowsUp",
```

Comment:
@neykov in `WAIT_FOR_SSHABLE` and `WAIT_FOR_WINRM_AVAILABLE` it is used that way, the use case is when set a non null default value and if you want to override it with false.
It skips the retries if null as well.